test/parallel/test-zlib-failed-init.js

```diff
@@ -11,9 +11,13 @@ const zlib = require('zlib');
 // no such rejection which is the reason for the version check below
 // (http://zlib.net/ChangeLog.txt).
 if (!/^1\.2\.[0-8]$/.test(process.versions.zlib)) {
-  assert.throws(() => {
-    zlib.createDeflateRaw({ windowBits: 8 });
-  }, /^Error: Init error$/);
+  common.expectsError(
+    () => zlib.createDeflateRaw({ windowBits: 8 }),
+    {
+      code: 'ERR_ZLIB_INITIALIZATION_FAILED',
```

Comment:
We need a new way to test this error once https://github.com/nodejs/node/pull/16511 is merged.